src/CabanaPD_Input.hpp

```diff
@@ -110,10 +113,16 @@ class Inputs
         else if ( inputs.contains( "low_corner" ) &&
                   ( inputs.contains( "high_corner" ) ) )
```

Comment:
It seems we have here extra parentheses: 
"( inputs.contains( "high_corner" ) ) )" could be "inputs.contains( "high_corner" ) )"